.github/workflows/update-cow-dependencies.yaml

```
@@ -0,0 +1,70 @@
+name: Update CoW Dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 00:00 on Wednesday
+    - cron: "0 0 * * 3"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v2
+        with:
+          ref: main
+          fetch-depth: 0
+
+      - name: Get latest release tag from CoW protocol services
+        id: latest-tag
+        run: |
+          TAG=$(curl -s https://api.github.com/repos/cowprotocol/services/releases/latest | jq -r '.tag_name')
+          echo "TAG=$TAG" >> $GITHUB_ENV
+
+      - name: Create branch
+        run: |
+          git checkout -b update-cow-dependencies/${{ env.TAG }}
+
+      - name: Update Cargo.toml
+        run: |
+          sed -i '/git = "https:\/\/github.com\/cowprotocol\/services.git"/s/tag = "v[0-9]*\.[0-9]*\.[0-9]*"/tag = "${{ env.TAG }}"/g' ./Cargo.toml
```

Updating the dependencies of the solvers can be a bit awkward when you actually have breaking changes that need to make it into the release for it to work. Usually what happens somebody creates a services version like `vsolvers-version` and uses that until the next time we need to update the version. Currently the replacement would not work for these situations.
Since this script always runs **after** a new version with the latest changes got cut I think it should be fine to also replace those named version tags here.

```suggestion
          sed -i '/git = "https:\/\/github.com\/cowprotocol\/services.git"/s/tag = ".*"/tag = "${{ env.TAG }}"/g' ./Cargo.toml
```

---

Completely forgot about those tags, thanks, fixed.